go/tasks/pluginmachinery/k8s/mocks/plugin_cleanup_policy.go

```diff
@@ -0,0 +1,30 @@
+// Code generated by mockery v1.0.1. DO NOT EDIT.
```

Comment:
I matched the version of Mockery used by the other interfaces in this repo, but it's definitely pretty out of date

Reply:
ya, so the way to generate the code is, 
`make download_tooling`, `make generate`

Reply:
thank you! 🙏 I've applied the changes.